src/views/Settings.vue

```diff
@@ -329,7 +345,8 @@ export default {
         state.pnp.peerConnectionStatus === PEER_CONNECTED,
       edgePeerId: state => state.pnp.remotePeerId,
       peerFetch: state => state.pnp.peerFetch,
-      version: state => state.version
+      version: state => state.edgeDevice.version,
```

Comment:
@vickywane I can see how the existing code confused you. It incorrectly showed the UI App version on the Edge Device card. The new code that you committed here fixes that by replacing `state.version` with `state.edgeDevice.version`. Let's update the corresponding description label next to the version number to clearly state that this is the edge device version.